airflow/kubernetes/kube_config.py

```diff
@@ -0,0 +1,218 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Module which parses the airflow.cfg for the Kubernetes related sections.
+"""
+import json
+
+from airflow import settings
+from airflow.configuration import conf
+from airflow.exceptions import AirflowConfigException
+
+
+class KubeConfig:  # pylint: disable=too-many-instance-attributes
```

Comment:
+1 for separating this out

Reply:
Yeah I saw you had started that in #5256